quill-finagle-postgres/src/main/scala/io/getquill/context/finagle/postgres/FinaglePostgresDecoders.scala

```diff
@@ -66,10 +66,7 @@ trait FinaglePostgresDecoders {
     case v: Double => v.toFloat
     case v: Float  => v
   }
-  implicit val doubleDecoder: Decoder[Double] = decoder[Double] {
-    case v: Double => v
-    case v: Float  => v.toDouble
```

Comment:
I've done all possible ways to reproduce that but couldn't make second match happens, most likely driver already does that, hence removed that part